BCStudentSoftwareDevTeam/celts
# Student admin

Did: fixes #984 and added a new role called student Admin. Student admin should have the same privileges as program managers for all programs aside from Bonner. 

Test: 
- Switch to branch studentAdmin
- Reset the data to ensure the role in the database (test data).
- In the bottom left corner switch the current user to Student Admin: bledsoef
- Go through events, create, delete, and edit events. Make sure the user has access and can edit for events associated with all programs except for Bonner 
- Switch to user to Admin and go to settings and then user management. Try to add and remove a new student Admin to make sure it works as expected.
-   review code
